mmdeploy/pytorch/ops/layer_norm.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+from torch import Tensor
+
+from mmdeploy.core import FUNCTION_REWRITER
+from mmdeploy.utils import Backend
+
+
+class LayerNormop(torch.autograd.Function):
```

Comment:
[this](https://github.com/pytorch/pytorch/blob/b369b89f235f54bc9de85d768fb62ac4579681dc/torch/onnx/symbolic_opset9.py#L1503) is the symbolic of `layer_norm` in pytorch. Rewrite it with SYMBOLIC_REWRITER might be a better choice (which can also support `torch.nn.functinoal.layer_norm`).

Reply:
Done, thanks for your [function definition](https://github.com/pytorch/pytorch/blob/b369b89f235f54bc9de85d768fb62ac4579681dc/torch/onnx/symbolic_opset9.py#L1503).

Yeasterday I have tried, but in vain. Because of missing this file.